DIRECTORY.md

```diff
@@ -30,6 +30,7 @@
   * Searching
     * [Binary Search](https://github.com/TheAlgorithms/Rust/blob/master/src/searching/binary_search.rs)
     * [Binary Search Recursive](https://github.com/TheAlgorithms/Rust/blob/master/src/searching/binary_search_recursive.rs)
+    * [Closest Points](https://github.com/TheAlgorithms/Rust/blob/master/src/searching/closest_points.rs)
```

Comment:
@imp2002 I don't think we still need DIRECTORY.md, do we?

Reply:
why? Is not need DIRECTORY.md or action?

Reply:
I think we can remove both, but let's discuss that on The Algorithms Discord server
No need to do that change in this PR